blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -1,10 +1,8 @@
 ---
-title: 'Simplifying Firmware Updates in Dasharo'
-abstract: "Updating your firmware is an important part of keeping your device
-          secure and reliable. Making this process as easy and reliable as
-          possible is, therefore, a big focus for Dasharo. In this article we'll
```

Comment:
Please revert the deletion of the first two sentences. @mkopec

Reply:
done 8bfad9d51ab588868e68877d4bb16a3fc3b31a43